src/components/pages/deployments/deployments.scss

```diff
@@ -5,5 +5,11 @@
 @import 'src/styles/themes';
 
 .deployments-page-container {
-   // TODO
+  display: flex;
+  flex-flow: column nowrap;
+  padding: $baseContentPadding;
+
+  .deployments-title {
```

Comment:
We're using a standard component for the page title. Why does it need to be restyled? Shouldn't the shared one be styled correctly to begin with? Something like the spacing around the page title should be consistent.

Reply:
done